modules/core/src/main/scala/org/tessellation/modules/Storages.scala

```diff
@@ -25,10 +27,12 @@ object Storages {
   def make[F[_]: Async: KryoSerializer: Supervisor](
     sdkStorages: SdkStorages[F],
     snapshotConfig: SnapshotConfig,
-    maybeRollbackHash: Option[Hash]
+    maybeRollbackHash: Option[Hash],
+    trustUpdates: Option[InternalTrustUpdateBatch]
   ): F[Storages[F]] =
     for {
       trustStorage <- TrustStorage.make[F]
+      _ <- trustUpdates.fold(Applicative[F].unit)(trustStorage.updateTrust(_))
```

Comment:
Not sure if this should be done here as a part of general `Storages` initialization flow. Maybe it would be better to have an additional `make(trustUpdates: Option[InternalTrustUpdateBatch]): F[TrustStorage]` method in `TrustStorage`. And the trust updates would happen as a part of `TrustStorage` initialization. WDYT?

Reply:
I agree with @TheMMaciek

Reply:
Added the third `make`.